packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts

```diff
@@ -45,6 +45,18 @@ export const runEvaluationJob = async (job: Job<RunEvaluationJobData>) => {
     }
 
     await progressTracker.incrementErrors()
+  } finally {
     await progressTracker.decrementTotal()
```

Comment:
decrementTotal was only the catch but I think is something we want to do always no?

Reply:
hmm no we only wanna do it when there's an error, if we succeed we don't wanna decrement the total